tests/api_test.py

```diff
@@ -2118,6 +2103,7 @@ def f(x):
     self.assertNotIn('constant.2 = f32[2]{0} constant({7, 14})', hlo_lines)
 
   def test_omnistaging_flag(self):
```

Comment:
delete?

Reply:
I have a superstition against deleting tests, in part because they leave a breadcrumb trail. But in this case I can delete them if you prefer.

Reply:
Keeping it around is fine

Reply:
You're right though: in this case it seems not so useful. I deleted it.